setup.cfg

```diff
@@ -55,6 +55,7 @@ whitelist_externals =
     pytest
 
 setenv =
+    PYTHONPATH = .
```

Comment:
The source codes are placed in the project root, why do we need patch PYTHONPATH?

Reply:
Otherwise, when running through tox, test imports do not work. You need to specify the python root directory.